### ISSUE TITLE
Cannot upgrade due to locked dependency

### ISSUE THREAD

**Issue by nayaabkhan** [erikras/redux-form#2338]
This change has caused an issue with installation of `redux-form` with latest peer dependencies: https://github.com/erikras/redux-form/commit/c790172c56ba308f09724fa62947fe1dff9c16f8#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R108

We are getting this as a result:
```
npm ERR! peerinvalid The package react-redux@4.4.6 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer redux-form@6.4.1 wants react-redux@4.3.0 || ^5.0.0-beta
```

A couple of people have already pointed the issue out on the commit.

**Comment by khizar** [comment]
So its working for now. The warning is still there, i cant understand why.

> npm WARN redux-form@6.4.1 requires a peer of react-redux@4.3.0 || ^5.0.0-beta but none was installed.

But its working for now without any problems if anyone comes here from a google search. will post if see any.

**Comment by daviscabral** [comment]
I am using `react-redux@4.4.6` and getting this warning as well.

**Comment by daviscabral** [comment]
@gustavohenke do you know if there was any special reason for that version lock?

**Comment by gustavohenke** [comment]
[Probably not](https://github.com/erikras/redux-form/pull/2331#issuecomment-269077543).

**Comment by erikras** [comment]
Published fix in [`v6.4.2`](https://github.com/erikras/redux-form/releases/tag/v6.4.2).
